content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md

```diff
@@ -0,0 +1,92 @@
+### INTRODUCTION
+Technology is fast growing in the world today. This recency in growth is enhancing the livelihoods of humans and their environment. This includes the building of many self-driven cars and some smart digital systems.
+A few problems in the understanding of its working algorithm have arisen. 
+   
+   These problems have triggered developers to ask questions like: "
+
```

Comment:
We can have a more briefer intro.

Give an insight of what the reader is going to learn, what is this topic about, why should they learn, see if beginners can understand this article. And, don' try to beat around the bush, directly get to the point.

Have a look at other articles to get an idea of how intro could be